### PR TITLE
Fix expm1 inaccuracies on complex inputs with small absolute values. Add Cosm1.

### PR DESCRIPTION
**Pull request by pearu**
As in the title.

Tests and improvement reports are in https://github.com/google/jax/pull/20144.

Accuracy tests are enabled in https://github.com/google/jax/pull/20436

### CODE REVIEW

**Comment by ddunl** [comment]
Double-checking - we can wait to merge this until after https://github.com/google/jax/pull/20144 right? Or is this a situation where we need to submit this first and then merge the JAX change shortly after

**Comment by pearu** [comment]
Landing https://github.com/google/jax/pull/20144 first should fix the CI failures in this PR.

**Comment by ddunl** [comment]
Great! Feel free to tag me after the JAX PR lands and I'll approve + land this one. Thanks again!!

**Review by hawkinsp — APPROVED** [review]

**Review by ddunl — APPROVED** [review]

**Pull request closed without merging**
